### PR TITLE
added missed identities

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10026

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/identity-protection/access-control/special-identities.md [hunk 1]
```diff
@@ -109,6 +121,18 @@ Any user who accesses the system through a sign-in process has the Authenticated
 |Default Location in Active Directory |cn=WellKnown Security Principals, cn=Configuration, dc=\<forestRootDomain\>|
 |Default User Rights| [Access this computer from the network](/windows/device-security/security-policy-settings/access-this-computer-from-the-network): SeNetworkLogonRight<br>  [Add workstations to domain](/windows/device-security/security-policy-settings/add-workstations-to-domain): SeMachineAccountPrivilege<br>  [Bypass traverse checking](/windows/device-security/security-policy-settings/bypass-traverse-checking): SeChangeNotifyPrivilege|
 
+## Authentication Authority Asserted Identity
+
+
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/special-identities.md:126]:
```suggestion
```

File: windows/security/identity-protection/access-control/special-identities.md [hunk 2]
```diff
@@ -121,6 +145,18 @@ Any user or process that accesses the system as a batch job (or through the batc
 |Default Location in Active Directory |cn=WellKnown Security Principals, cn=Configuration, dc=\<forestRootDomain\>|
 |Default User Rights| none|
 
+## Console Logon
+
+
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/special-identities.md:150]:
```suggestion
```

File: windows/security/identity-protection/access-control/special-identities.md [hunk 3]
```diff
@@ -197,6 +233,18 @@ Membership is controlled by the operating system.
 |Default Location in Active Directory |cn=WellKnown Security Principals, cn=Configuration, dc=\<forestRootDomain\>|
 |Default User Rights| [Access this computer from the network](/windows/device-security/security-policy-settings/access-this-computer-from-the-network): SeNetworkLogonRight</br> [Act as part of the operating system](/windows/device-security/security-policy-settings/act-as-part-of-the-operating-system): SeTcbPrivilege</br> [Bypass traverse checking](/windows/device-security/security-policy-settings/bypass-traverse-checking): SeChangeNotifyPrivilege|
 
+## Fresh public key identity
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/special-identities.md:236]:
```suggestion
## Fresh Public Key Identity
```

File: windows/security/identity-protection/access-control/special-identities.md [hunk 4]
```diff
@@ -209,6 +257,30 @@ Any user who is logged on to the local system has the Interactive identity. This
 |Default Location in Active Directory |cn=WellKnown Security Principals, cn=Configuration, dc=\<forestRootDomain\>|
 |Default User Rights| None|
 
+## IUSR
+
+
+Internet Information Services (IIS) use this account by default whenever anonymous authentication is enabled.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/special-identities.md:263]:
```suggestion
Internet Information Services (IIS) uses this account by default whenever anonymous authentication is enabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MaratMussabekov, kindly commit the few minor suggested changes. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @MaratMussabekov @JohanFreelancer9

**Pull request merged**